src/lightning_app/cli/commands/connection.py

```diff
@@ -129,17 +130,20 @@ def connect(app_name_or_id: str, yes: bool = False):
                         retriever.app_id,
                         target_file=target_file,
                     )
-                    click.echo(f"Storing `{command_name}` under {target_file}")
-                    click.echo(f"You can review all the downloaded commands under {commands_folder} folder.")
+                    pretty_command_name = command_name.replace("_", " ")
```

Comment:
Lots of repetition of this line for cleaning the command name. Don't think it should be addressed here, just a note for future us haha

Reply:
Yeah, we should probably clean up a bit the logic around it. I want this change to get into the release. So adding this change too in this PR.